packages/monitoring-system/data-processor/test/unit/data-processors/cloud-logs-data-processor.spec.ts

```diff
@@ -152,44 +172,48 @@ describe('Cloud Logs Processor', () => {
       );
     });
 
+
     describe('correctly formed incoming log messages', () => {
       beforeEach(() => {
         processor = new CloudLogsProcessor({
           firestore: mockFirestore,
           subscription: mockSubscription,
+          listenLimit: LISTEN_LIMIT_SECS,
         });
       });
 
+
       describe('execution start and execution end logs', () => {
         // what the execution record would look like after receiving just the start log
         const executionRecordStart = {
           document: {
             '4ww4alqs7ikq': {
               execution_id: '4ww4alqs7ikq',
-              bot_id: 'merge_on_green',
-              start_time: 1595536893000,
+              bot_name: 'merge_on_green',
```

Comment:
I changed the field `bot_id` to `bot_name` to make it more apparent that the primary key is the name